docs/source/docker/getting_started/hardware_recommendations.rst

```diff
@@ -0,0 +1,82 @@
+.. _ref-hardware-recommendations:
+
+Hardware recommendations
+========================
+
+Usually, the Lightly worker is run on a cloud machine,
+which is spin up specifically to run the docker.
+Our recommendations on the hardware configuration of this machine are
+based on three criteria:
+
+- speed: The worker should process your dataset as quickly a possible.
+- cost-effectiveness: The machine should be cheap to rent.
+- stability: The worker should not crash because it runs out of memory.
+
+As a starting point, we recommend a machine with the following configuration:
+
+- 8 vCPUs
+- 30-32 GB of system memory
+- a NVIDIA-GPU with 16GB of video memory, e.g. the T4 GPU.
+
```

Comment:
We should give somewhere here clear recommendations for different use cases:

E.g. something like this:

I want to process up to 500'000 images using the Lightly worker:
- we recommend to use an AWS `g4dn.xlarge` or equivalent with 4 cores, 16 GB of system memory, T4 GPU

I want to process up to 5'000'000 images using the Lightly worker:
- we recommend to use an AWS `g4dn.2xlarge` or equivalent with 8 cores, 32 GB of system memory, T4 GPU


I want to process up to 100 videos:
- we recommend to use an AWS `g4dn.2xlarge` or equivalent with 8 cores, 32 GB of system memory, T4 GPU

I want to process up to 50'000 videos using the Lightly worker:
- we recommend to use an AWS `g4dn.4xlarge` or equivalent with 16 cores, 64 GB of system memory, T4 GPU

Reply:
I merged the frame/video parts as the both depend on the number of images or frames.